app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1>User Details</h1>
```

Comment:
- [ ] Kindly create the index page for the users `index` action

Reply:
Created index file as per instructions